src/js/components/RadioButton/README.md

```diff
@@ -166,6 +166,16 @@ Defaults to
 100%
 ```
 
+**radioButton.check.size**
+
+The check size of the RadioButton. Expects `string`.
```

Comment:
Should this be `number` instead of `string`?

Reply:
Yeah, `number` makes more sense than `string`. thanks.